index.Rmd

```diff
@@ -14,29 +14,60 @@ library(rgdal)
 library(broom)
 ```
 
-# Preface {-}
+# Preface and motivation {-}
 
-This resource is an introduction to the R programming language. It can be used as standalone learning tool, or as part of training sessions.
+This resource is an introduction to the R programming language, which is a tool developed by statisticians across the world to do statistics, data analytics, create graphics, make dashboards and write reports. This online book can be used as standalone learning tool, or as part of training sessions.
 
-It takes you through from the first steps of opening RStudio through using it for visualisations such as graphs and maps - basically an introduction to using R for simple but effective data management, manipulation, calculations, and visualisation.
+<font size="5"><b style="color:#000000";>Chapters covered:</b></font> 
 
-It does however work on the assumption that [R and R Studio](https://www.rstudio.com/products/rstudio/download/) are already installed.
+1. [Getting started with RStudio](getting-started.html)
+2. [Downloading data for tutorial](data.html)
+3. [General Coding](general-coding.html)
+4. [Loading and exploring data](loading-and-exploring-data.html)
+5. [Data manipulation with **`dplyr`**](dplyr.html)
+6. [Data visualisation with **`ggplot2`**](graphs.html)
+7. [Statistical operations](statistical-operations.html)
+8. [Iteration](iteration.html)
+9. [Functions](functions.html)
+10. [Maps](maps.html)
+11. [RMarkdown](r-markdown.html)
+
+\n
+
+<font size="5"><b style="color:#000000";>The art of the possible - show me what R is capable of</b></font>
+
+The below links hopefully motivate some of the possible thing we can do with R, the first two are built in house at DfE. 
```

Comment:
Want to make this more comprehensive and collaborative. R isn't just for visualizations which is what is coming off here. 

Think we need to cover:
1. Statistical Modelling
2. Visualisation - Charts, Interactive visualistations, maps etc.
3. R Markdown and sub strands - reproducible docs, interactive reports, interactive dashboards.
4. R Shiny Apps - and that we are building the capability for hosting them internally
5. A host of other things - websites (this book), blogs....

We then want to give some more collaborative examples accross the department rather than juts show the exclusion ones. I'd probably link to rstudio's rmarkdown and shiny pages too.